drivers/filesystems/fastfat/fsctl.c

```diff
@@ -811,7 +817,14 @@ VfatMount(
     {
         /* Cleanup */
         if (DeviceExt && DeviceExt->FATFileObject)
-            ObDereferenceObject (DeviceExt->FATFileObject);
+        {
+            LARGE_INTEGER Zero = {{0,0}};
+            CcUninitializeCacheMap(DeviceExt->FATFileObject,
+                                   &Zero,
```

Comment:
or NULL

Reply:
Strangely, not the same thing.

Reply:
Thanks for the explanation, it was not completely clear from the documentation.